Resources/views/Form/javascript.js.twig

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function () {
+  $('select2 select_ajax').click(function (e) {
+    if ($(this).data('url') == '') {
+      throw 'url is not defined'
```

Comment:
you should pass the url in some way through data-url that several widget can attack different backends